rust/perspective-viewer/src/rust/components/column_settings_sidebar/style_tab.rs

```diff
@@ -204,19 +204,21 @@ pub fn StyleTab(p: &StyleTabProps) -> Html {
                 NumberColumnStyleDefaultConfig,
             >(&renderer, &column_name, ty)
             .map(|(config, default_config)| {
-                let on_change = Callback::from(move |config| {
-                    send_config(
-                        &renderer,
-                        &presentation,
-                        view.clone(),
-                        column_name.clone(),
-                        config,
-                    );
-                });
+                let on_change = {
+                    clone!(column_name, view);
+                    Callback::from(move |config| {
+                        send_config(
+                            &renderer,
+                            &presentation,
+                            view.clone(),
+                            column_name.clone(),
+                            config,
+                        );
+                    })};
                 html_template! {
                     <div class="item_title">{title.clone()}</div>
                     <div class="style_contents">
-                        <NumberColumnStyle  { config } {default_config} {on_change} />
+                        <NumberColumnStyle column_name={column_name.clone()} view={view.clone()}  { config } {default_config} {on_change} />
```

Comment:
Don't pass bind and pass `View` - pass the state object `Session` and request the view immediately where needed. This doesn't matter as much for this change due to the ephemeral nature of this component, but for components that persist between query changes, `View` can not only go stale, any remaining stale references will _segfault_ if you try to call them.